### DISCUSSION TITLE
Help with Chia Installation on Debian PPC64

### DISCUSSION THREAD

**Discussion by WMark77** [Chia-Network/chia-blockchain#6195] [Support]
Hello,

I'm trying to install Chia in Debian PPC64 (Big Endian).  Script installation (sh install.sh) runs fine untill reaches "Building wheel for chiavdf (PEP 517)" stage:

  Building wheel for chiavdf (PEP 517) ... **error**
  ERROR: Command errored out with exit status 1:
   command: /opt/chia-blockchain/venv/bin/python /opt/chia-blockchain/venv/lib/python3.9/site-packages/pip/_vendor/pep517/in_process/_in_process.py build_wheel /tmp/tmpdwavdo99                                                                        
       cwd: /tmp/pip-install-owz4wda4/chiavdf_f3caa94d51da49ada32b4aaca84a7038                                              
  Complete output (87 lines):                                                                                               
  running bdist_wheel                                                                                                       
  running build                                                                                                             
  running build_ext                                                                                                         
  -- The C compiler identification is GNU 10.2.1                                                                            
  -- The CXX compiler identification is GNU 10.2.1                                                                          
  -- Detecting C compiler ABI info                                                                                          
  -- Detecting C compiler ABI info - done                                                                                   
  -- Check for working C compiler: /usr/bin/cc - skipped                                                                    
  -- Detecting C compile features                                                                                           
  -- Detecting C compile features - done                                                                                    
  -- Detecting CXX compiler ABI info                                                                                        
  -- Detecting CXX compiler ABI info - done                                                                                 
  -- Check for working CXX compiler: /usr/bin/c++ - skipped                                                                 
  -- Detecting CXX compile features                                                                                         
  -- Detecting CXX compile features - done                                                                                  
  GMP_INCLUDES=/usr/include/powerpc64-linux-gnu                                                                             
  -- Found GMP: /usr/include/powerpc64-linux-gnu (Required is at least version "5.1.0")                                     
  GMP_INCLUDES=/usr/include/powerpc64-linux-gnu                                                                             
  -- Found GMPXX: /usr/lib/powerpc64-linux-gnu/libgmpxx.so                                                                  
  -- pybind11 v2.6.2                                                                                                        
  -- Found PythonInterp: /opt/chia-blockchain/venv/bin/python (found version "3.9.2")                                       
  -- Found PythonLibs: /usr/lib/powerpc64-linux-gnu/libpython3.9.so                                                         
  -- Performing Test HAS_FLTO                                                                                               
  -- Performing Test HAS_FLTO - Success                                                                                     
  -- Configuring done                                                                                                       
  -- Generating done                                                                                                        
  -- Build files have been written to: /tmp/pip-install-owz4wda4/chiavdf_f3caa94d51da49ada32b4aaca84a7038                   
  Scanning dependencies of target verifier_test                                                                             
  Scanning dependencies of target chiavdf                                                                                   
  [ 16%] Building CXX object CMakeFiles/verifier_test.dir/verifier_test.cpp.o                                               
  [ 33%] Building C object CMakeFiles/verifier_test.dir/refcode/lzcnt.c.o                                                   
  [ 50%] Building CXX object CMakeFiles/chiavdf.dir/python_bindings/fastvdf.cpp.o                                           
  [ 66%] Building C object CMakeFiles/chiavdf.dir/refcode/lzcnt.c.o                                                         
  [ 83%] Linking CXX executable verifier_test                                                                               
  /usr/bin/ld: CMakeFiles/verifier_test.dir/verifier_test.cpp.o: in function `Reducer::run()':                              
  verifier_test.cpp:(.text._ZN7Reducer3runEv[_ZN7Reducer3runEv]+0x13c): undefined reference to `lzcnt64_hard'               
  /usr/bin/ld: verifier_test.cpp:(.text._ZN7Reducer3runEv[_ZN7Reducer3runEv]+0x214): undefined reference to `lzcnt64_hard'  
  /usr/bin/ld: verifier_test.cpp:(.text._ZN7Reducer3runEv[_ZN7Reducer3runEv]+0x2ec): undefined reference to `lzcnt64_hard'  
  collect2: **error**: ld returned 1 exit status                                                                                
  gmake[2]: *** [CMakeFiles/verifier_test.dir/build.make:120: verifier_test] **Error 1**                                        
  gmake[1]: *** [CMakeFiles/Makefile2:117: CMakeFiles/verifier_test.dir/all] **Error 2**                                        
  gmake[1]: *** Waiting for unfinished jobs....                                                                             
  [100%] Linking CXX shared module build/lib.linux-ppc64-3.9/chiavdf.cpython-39-powerpc64-linux-gnu.so                      
  [100%] Built target chiavdf                                                                                               
  gmake: *** [Makefile:103: all] **Error 2**                                                                                    
  Traceback (most recent call last):                                                                                        
    File "/opt/chia-blockchain/venv/lib/python3.9/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 280, in <module>               

Is there any possibility of installing Chia on the PPC64 Big -Endian, even if it is compiling binary by binary?

Thank you in advance.

**Comment by BrandtH22** [comment]
@WMark77 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - the install from source instructions can be found here https://docs.chia.net/installation#from-source

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
